### PR TITLE
fixed a bug where the save slot menu would show up 

### PR DESCRIPTION
**Pull request by andrewhong04**
...when you would specify a save slot to load via argparser

also removed an unused import

### CODE REVIEW

File: tuxemon/states/persistance/load_menu.py [hunk 1]
```diff
@@ -32,8 +32,7 @@ def on_menu_selection(self, menuitem: Optional[MenuItem[None]]) -> None:
                 self.client.pop_state(old_world)
             except ValueError:
                 # when game is loaded from the start menu
-                self.client.pop_state()  # close this menu
```
Comment (bitcraft) [inline tuxemon/states/persistance/load_menu.py:35]: why are these removed?

Comment (andrewhong04) [reply tuxemon/states/persistance/load_menu.py:35]: > why are these removed?

The game seemed to work without those functions when I added the new solution. There wasn't any difference with having them removed, so I'll add them back right now.

**Review by bitcraft — APPROVED** [review]
thank you!
